src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -345,6 +345,16 @@
             <ui:InCanvasSearchControl RequestShowInCanvasSearch="ShowHideInCanvasControl" />
         </Popup>
 
+        <Popup Name="NodeAutoCompleteSearchBar"
+               StaysOpen="True"
+               AllowsTransparency="True"
+               IsOpen="False"
+               Placement="MousePoint"
+               HorizontalOffset="-290"
+               DataContext="{Binding NodeAutoCompleteSearchViewModel}">
+            <ui:NodeAutoCompleteSearchControl RequestShowNodeAutoCompleteSearch="ShowHideInCanvasControl" />
```

Comment:
Based on your comments above, shouldn't this be `ShowHideNodeAutoCompleteControl`?

Reply:
@aparajit-pratap Yup, let me update